src/Microsoft.Health.Fhir.Tests.Common/TestFiles/R4/Bundle-TransactionForRollBack.json

```diff
@@ -0,0 +1,43 @@
+﻿{
+    "resourceType": "Bundle",
+    "id": "bundle-transaction",
+    "meta": {
+        "lastUpdated": "2014-08-18T01:43:30Z"
+    },
+    "type": "transaction",
+    "entry": [
+    	 {
+            "fullUrl": "urn:uuid:61ebe359-bfdc-4613-8bf2-c5e300945f0a",
+            "resource": {
+                "resourceType": "Patient",
+                "text": {
+                    "status": "generated",
+                    "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Some narrative</div>"
+                },
+                "active": true,
+                "name": [
+                    {
+                        "use": "official",
+                        "family": "ADHI",
+                        "given": [
+                            "Peter",
+                            "Jenny"
+                        ]
+                    }
+                ],
+                "gender": "male",
+                "birthDate": "1974-12-25"
+            },
+            "request": {
+                "method": "POST",
+                "url": "Patient"
+            }
+        },
+       {
+            "request": {
+                "method": "GET",
+                "url": "Patient/12345"
```

Comment:
Are we relying on the fact that there won't be a `Patient/12345` in the database to cause the rollback?

Reply:
Yes. I am deleting all the existing patient resources before executing a new test.